src/main/assets/scss/main.scss

```diff
@@ -1 +1,7 @@
 @import 'govuk-frontend';
+
+.spacer {
```

Comment:
Use the GOV UK spacer, don't add in a custom one. They exist already within the gov uk frontend 

https://design-system.service.gov.uk/styles/spacing/

Reply:
changed